components/card/card_default.story.vue

```diff
@@ -18,7 +18,10 @@
         aria-label="Menu button"
       >
         <template #icon>
-          <icon-menu-vertical />
+          <dt-icon
+            name="more-vertical"
+            size="100"
+          />
```

Comment:
Good for now (including in IVR recipes). I've logged a task to evaluate size later. [DT-814](https://dialpad.atlassian.net/browse/DT-814).

No action required.